### PR TITLE
Add more status LEDs

### PR DESCRIPTION
**Pull request by bbrk24**
This PR adds more status LEDs, bringing us up to a total of six LEDs per capsule and two for the payload bay.

@TheRoyalIrishman This means we need seven each of 100- and 240-Ohm resistors. Do we have enough?

### CODE REVIEW

**Comment by bbrk24** [comment]
I don't have time to fix this right now, but I just remembered that the Arduino Micro runs at 5V and not 3.3V, so I may have to change the resistors in the payload bay. If nothing else, the LEDs should be connected to the +5V pin instead of the +3.3V pin.

File: sketch_oct9a.ino [hunk 1]
```diff
@@ -103,9 +103,16 @@ void updateStatusLEDs() {
...
   digitalWrite(7, good);
-  digitalWrite(10, !good);
 }
 
+void updateGPSLEDs() {
+  bool good = gps.getStatus() == GPS::ACTIVE;
+  digitalWrite(6, good);
+}
+
+// TODO: updateSDCardLEDs
```
Comment (bbrk24) [inline sketch_oct9a.ino]: Now that #7 has been merged I need to update this

File: SD_Card.h [hunk 2]
```diff
@@ -74,13 +163,19 @@ class SDCard {
...
-	}
+      if (!m_begun) {
+        m_begun = SD.begin(M_CHIP_SELECT);
+      }
+      
+      if (m_begun) {
+        if (!m_proven) {
+          m_proven = selfTest();
+        }
+
```
Comment (DarthSethious1) [inline SD_Card.h:168]: check to make sure that it is proven after we re-evaluate 'm_proven'

File: SD_Card.h [hunk 3]
```diff
@@ -4,16 +4,105 @@
...
+      /** The SD card is ready to write to. */
+      ACTIVE,
+    };
+
+    Status getStatus() {
+      if (!m_begun) {
+        return Status::NOT_CONNECTED;
+      }
+
+      // TODO: which should take precedence?
```
Comment (DarthSethious1) [inline SD_Card.h]: i like your current precedence

File: SD_Card.h [hunk 4]
```diff
@@ -4,16 +4,105 @@
...
+      if (!m_begun) {
+        return false;
+      }
+
+      // Close the file for now to open a new one
+      bool open = (bool)m_sdCardFile;
+      if (open) {
+        m_sdCardFile.close();
+      }
+
```
Comment (DarthSethious1) [inline SD_Card.h:29]: Very good test. well structured, commented, and written

**Review by DarthSethious1 — COMMENTED** [review]
I did not open and look at the .ms14 files because you've shown me them in person. I will approve once you have commented back on my comments, just so i know you've seen them

**Review by CClarke513 — APPROVED** [review]
Everything looks good to me

**Pull request merged**